default_adapter.go

```diff
@@ -679,11 +678,7 @@ func (me *DefaultAdapter) QuoteIdentifier(buf *SqlBuilder, ident IdentifierExpre
 
 //Generates SQL NULL value
 func (me *DefaultAdapter) LiteralNil(buf *SqlBuilder) error {
-	if buf.IsPrepared {
-		return me.PlaceHolderSql(buf, nil)
-	} else {
-		buf.Write(me.Null)
-	}
+	buf.Write(me.Null)
 	return nil
```

Comment:
I figured out that it was best to change it here instead, let me know if you disagree, I can revert the changes and start over. I am not sure but it could possibly affect specific drivers?

Thanks